appmap/_implementation/env.py

```diff
@@ -38,7 +39,7 @@ def __init__(self, env=None):
 
         output_dir = self.get("APPMAP_OUTPUT_DIR",
                                os.path.join('tmp', 'appmap'))
-        self._output_dir = os.path.abspath(output_dir)
+        self._output_dir = Path(os.path.abspath(output_dir))
```

Comment:
Can this use `Path.resolve` instead?

Reply:
Sure, good point.

Reply:
I've split env cleanup into a separate commit and added a couple more changes, do you want to take a look? https://github.com/applandinc/appmap-python/pull/111/commits/ede8e19fb5c5cfe2bd5eedd67c35d5cd65916d21

Reply:
Looks good, I'd say this is ready to go.